### ISSUE TITLE
Portainer image pull correction.

### ISSUE THREAD

**Issue by shadowsbane0** [kelinger/OmniStream#5]
The image line fails to pull Portainer. Corrected to “portainer/portainer-ce”. Works now. I also looked on Docker Hub and portainer/portainer-ce is the new branch with portainer/portainer being end of life and depreciated.

**Comment by TechPerplexed** [comment]
@shadowsbane0 as a side note: I know I mentioned this in the Gooby place as well, but you can't just update your containers. We haven't even written the documentation yet... but you'll need to rename your container to 801 or higher, otherwise it will just be overwritten the next time you pull the project with `omni refresh` etc.

**Comment by shadowsbane0** [comment]
Thanks @TechPerplexed. I saw that and mentioned it one of the other issue posts. Thanks for the update. I’ll work on renaming the corrections. I’m still working on the rewrites of my Readarr, Calibre, and Calibre-Web yaml’s to work with the Omni ecosystem.

**Comment by TechPerplexed** [comment]
Nice! Funny enough we discussed Readarr and Mylarr yesterday and agreed those would make good additions to Omni, so if you don't want to rewrite your own container, you can just wait for the official one :)

**Comment by shadowsbane0** [comment]
I’ll wait for the official version. Readarr isn’t hard. Calibre is the PITA. It doesn’t play well with having its db on GSuite. I was keeping it local in the Downloads mount along with all of my recycle bins for the other “Arr” services and the complete/incomplete directories for Deluge. I recently saw one of your instruction manuals that reference the same setup.

I’m happy to test things out if you want to try them. Part of why I chose to play with BTRFS is because I can snapshot the system before trying a bunch of new things. 

I’m on hold at the moment with anything other than this while I wait for the next week to go by while my content is synched from GSuite to local. I’ve got a Gig fiber connection and it still takes forever! I’m six days in and just under half of the content synched.

Have any questions on my setup my email is in my profile. I consider it a junk email account though not everything there is junk. It is an account I don’t care if its public. I started using it long before the days of email alias’s became a thing. Back when I corned the market on the handle at every free email service that existed. :}
